packages/compass-aggregations/src/components/stage-editor/stage-editor.jsx

```diff
@@ -90,6 +94,20 @@ export class StageEditor extends PureComponent {
     this.props.onChange(this.props.index, value);
   };
 
+  onBlur = () => {
+    const value = this.editor?.getValue();
+    if (value !== this.initialValue) {
```

Comment:
tiny NIT: value can be different from the initialValue also if editor would not be defined

```suggestion
    if (value !== undefined && value !== this.initialValue) {
```

Reply:
thanks! yes, it can be. i added a guard.